dev/archery/archery/crossbow/cli.py

```diff
@@ -156,8 +156,13 @@ def submit(obj, tasks, groups, params, job_prefix, config_path, arrow_version,
         click.echo('Pushed job identifier is: `{}`'.format(job.branch))
 
 
+# Get the default branch name from the repository
+arrow_source_dir = ArrowSources.find()
+repo = Repo(arrow_source_dir.path)
```

Comment:
Can we avoid doing this at module import?

Reply:
Yes, thank you for this suggestion, I've moved the code to set the default within the function.